lib/bloc/podcast_payments/podcast_payments_bloc.dart

```diff
@@ -335,9 +335,9 @@ class PodcastPaymentsBloc with AsyncActionsHandler {
     tlv["episode"] = episode.title;
     tlv["action"] = boost ? "boost" : "stream";
     tlv["time"] = _formatDuration(position.position);
-    tlv["feedID"] = _getPodcastIndexID(episode);
+    tlv["feedID"] = int.parse(_getPodcastIndexID(episode));
```

Comment:
There is a conversion inside the method `_getPodcastIndexID`. Maybe just omit it instead of the roundtrip?

Reply:
@ademar111190 please fix according to @roeierez suggestion, but make sure it does create a regression (if there are other calls that expect a string)

Reply:
Changes applied @kingonly @daveajones @roeierez 🎉